property_groups/dream_prompt.py

```diff
@@ -69,8 +69,8 @@ def model_options(self, context):
         case Pipeline.STABLE_DIFFUSION:
             def model_case(model, i):
                 return (
-                    model.model,
-                    os.path.basename(model.model).replace('models--', '').replace('--', '/'),
+                    model.model_base,
```

Comment:
This will allow an exported prompt to be more portable since it will just include the model name instead of a full path.

I had originally changed this to use the model's base path in the DirectML branch and didn't realize it regressed in the error checking branch. Interestingly it didn't break model snapshot selection due to a quirk in how `os.path.join()` will ignore arguments that come before a full path.

Reply:
Ah, good call